### PR TITLE
adding details on burning HNT for DCs to maker wallets

### PR DESCRIPTION
**Pull request by PharkMillups**

### CODE REVIEW

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+hide_title: true
+sidebar_label: Burning HNT to Maker Wallet
+slug: /mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Burning HNT to Your Maker Wallet
+
+Any [approved Hotspot Maker](/mine-hnt/approved-makers) is responsible for supplying Data Credits (DCs) to the onboarding server for their Hotspots. They do this by burning HNT to their
```
Comment (abhay) [inline docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]:
```suggestion
Any [approved Hotspot Maker](/mine-hnt/approved-makers) is responsible for supplying Data Credits (DCs) to the onboarding server for Hotspots they sell. They do this by burning HNT to their 
```

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+:::info 
+
+A full list of approved maker wallet addresses [can be seen via this API](https://onboarding.dewi.org/api/v2/makers).
+
+:::
+
+## Maker Wallet Address Integration
+
+As an approved Maker, you've been supplying the DeWi with the onboarding details for all the Hotspots you manufacture and distribute. In turn, the DeWi adds them to the onboarding server. When a Hotspot is onboarded by your end customer, the onboarding server knows to associate it with your Maker ID, and associated Maker Wallet address. If, when a Hotspot is onboarded, it's present in the onboarding server and you have ample Data Credits to pay the onboarding fees, the blockchain will add the Hotspot. And if you've [integrated your Hotspot in the Helium Mobile Wallet onboarding flow](/mine-hnt/hotspot-makers/hotspot-integration-testing), the Wallet knows to use your approved Maker Wallet address. At this point, the Data Credits will be flushed from your wallet address.
```
Comment (cokes518) [inline docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]: As an approved maker, you **will supply DeWi the onboarding** details...

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Burning HNT to Your Maker Wallet
+
+Any [approved Hotspot Maker](/mine-hnt/approved-makers) is responsible for supplying Data Credits (DCs) to the onboarding server for their Hotspots. They do this by burning HNT to their 
+specific maker address. 
+
+:::info 
+
+A full list of approved maker wallet addresses [can be seen via this API](https://onboarding.dewi.org/api/v2/makers).
```
Comment (abhay) [inline docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx:17]: Can we automatically generate a table with this API response?

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx (hunk 2 above)
Comment (cokes518) [reply docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]: > And if you've integrated your Hotspot in the Helium Mobile Wallet onboarding flow

Change to "And if you've integrated your Hotspot in the Hotspot App onboarding flow

Comment (abhay) [inline docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]:
```suggestion
As an approved Maker, you've been supplying the DeWi with the onboarding details for all the Hotspots you manufacture and distribute. In turn, the DeWi adds them to the onboarding server. When a Hotspot is onboarded by your end customer, the onboarding server knows to associate it with your Maker ID, and associated Maker Wallet address. If, when a Hotspot is onboarded, it's present in the onboarding server and you have ample Data Credits to pay the onboarding fees, the blockchain will add the Hotspot. And if you've [integrated your Hotspot in the Helium Mobile Wallet onboarding flow](/mine-hnt/hotspot-makers/hotspot-integration-testing), the Wallet knows to use your approved Maker Wallet address. At this point, the Data Credits will be used for the transactions with your wallet address as the payer. 
```

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+
+## Hotspot Data Credit Fees
+
+In order for any Hotspot to join the Helium blockchain, its maker address must have sufficient Data Credits. A full list of [Helium blockchain data credit fees can be found here](/blockchain/transaction-fees). Hotspot Makers need to account for at least `5,000.000` total Data Credits per Hotspot. Specifically, for each Hotspot, a Maker must supply:
+
+* `4,000,000` Data Credits for the **Add Hotspot** fee. This is a one-time fee.
+* `1,000,000` Data Credits for the **Assert Location** fee. Each Hotspot Maker is required to supply at least one **Assert Location* for their Hotspots. 
+
+As a maker, if you only the one-time **Add Hotspot** fee and one **Assert Location** fee, you'll supply `5,000.000` total Data Credits per Hotspot. Add `1,000,000` Data Credits for each additional **Assert Location** fee you with to supply.
```
Comment (abhay) [inline docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]: should we add some notes about the transaction fees as well?

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+:::warning 
+
+If your wallet address does not have enough DCs to pay for required Hotspot Transactions, they will fail. 
+
+:::
+
+
+## Maker Integration Test Wallet Address
+
+During the course of your testing, you can use the Maker Integration Test wallet address created by the DeWi.  Here is that address:
```
Comment (abhay) [inline docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]:
```suggestion
During the course of your testing, you can use the Maker Integration Test wallet address created by the DeWi.  This wallet does not contain any Data Credits so onboarding transactions will fail but it allows you to smoke test the onboarding steps. Here is that address:
```

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx (hunk 2 above)
Comment (cokes518) [reply docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]: > At this point, the Data Credits will be flushed from your wallet address. 

Change to "At this point, Data Credits will be consumed by the onboarding transactions (Add Hotspot and/or Assert Location) and deducted from your Maker wallet address.

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+As an approved Maker, you've been supplying the DeWi with the onboarding details for all the Hotspots you manufacture and distribute. In turn, the DeWi adds them to the onboarding server. When a Hotspot is onboarded by your end customer, the onboarding server knows to associate it with your Maker ID, and associated Maker Wallet address. If, when a Hotspot is onboarded, it's present in the onboarding server and you have ample Data Credits to pay the onboarding fees, the blockchain will add the Hotspot. And if you've [integrated your Hotspot in the Helium Mobile Wallet onboarding flow](/mine-hnt/hotspot-makers/hotspot-integration-testing), the Wallet knows to use your approved Maker Wallet address. At this point, the Data Credits will be flushed from your wallet address. 
+
+
+## Hotspot Data Credit Fees
+
+In order for any Hotspot to join the Helium blockchain, its maker address must have sufficient Data Credits. A full list of [Helium blockchain data credit fees can be found here](/blockchain/transaction-fees). Hotspot Makers need to account for at least `5,000.000` total Data Credits per Hotspot. Specifically, for each Hotspot, a Maker must supply:
```
Comment (cokes518) [inline docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]: Capitalize Blockchain

Comment (cokes518) [reply docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]: Capitalize Maker

**Review by abhay — DISMISSED** [review]
Small nits. Generally great though. Thanks for documenting.

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx (hunk 5 above)
Comment (cokes518) [inline docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]: Add:

This Test wallet address can be used until you have your own designated Maker address. Adding Hotspots with this address as `payer` will not add Hotspots to the blockchain and you should not burn Data Credits into this address.

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx (hunk 3 above)
Comment (PharkMillups) [reply docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx:17]: In browser? Or in the docs?

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx (hunk 6 above)
Comment (PharkMillups) [reply docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]: Fixed.

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx (hunk 2 above)
Comment (PharkMillups) [reply docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]: Fixed.

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx (hunk 4 above)
Comment (PharkMillups) [reply docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]: Yes. Added.

File: docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx (hunk 5 above)
Comment (PharkMillups) [reply docs/mine-hnt/hotspot-makers/burn-hnt-to-maker-wallet.mdx]: Fixed.

**Review by kent-williams — APPROVED** [review]

**Pull request merged**